src/ui/components/uicontroller/UiContainer.tsx

```diff
@@ -49,15 +57,33 @@ interface UiContainerProps {
    */
   bottom?: ReactNode;
   /**
-   * The components to be put in the ad slot.
+   * The components to be put in the ad slots.
+   *
+   * @remarks
+   * <br/> - Currently only supported for web.
    */
-  ad?: ReactNode;
+  ad?: AdUiContainer;
```

Comment:
Any particular reason why you went with `ad.top`, `ad.center` and `ad.bottom` instead of `adTop`, `adCenter` and `adBottom`?

Reply:
I think usage looks a bit weird: https://github.com/THEOplayer/react-native-theoplayer-ui/blob/2e7f0f55d57501912f9227b3cba91975e0b70e0d/example/App.tsx#L134-L161

I'd go for `adTop`, `adCenter` and `adBottom`.